src/pages/index.tsx

```diff
@@ -247,26 +239,22 @@ const Home: FunctionComponent = () => {
                                 src="/home/branded-icons/Code-insights-icon.svg"
                                 alt="Completions Brand Icon"
                             />
-                            <Heading size="h2" className="pb-4 pt-6 !-tracking-[1px] md:leading-10">
-                                Track trends in your codebase
-                            </Heading>
-                            <Heading size="h3" className="leading-[30px] !-tracking-[0.25px] text-gray-500">
+                            <h2 className="pb-4 pt-6">Track trends in your codebase</h2>
+                            <h3 className="text-gray-500">
                                 Transform your code into a queryable database to create custom dashboards in seconds.
-                            </Heading>
+                            </h3>
                         </div>
                         <div className="flex w-full flex-col rounded-2xl border-1 border-gray-200 bg-white px-10 py-16">
                             <img
                                 className="h-[48px] w-[48px]"
                                 src="/home/branded-icons/batch-changes-icon.svg"
                                 alt="Completions Brand Icon"
                             />
-                            <Heading size="h2" className="pb-4 pt-6 !-tracking-[1px] md:leading-10">
-                                Automate large-scale code changes
-                            </Heading>
-                            <Heading size="h3" className="leading-[30px] !-tracking-[0.25px] text-gray-500">
+                            <h2 className="pb-4 pt-6">Automate large-scale code changes</h2>
+                            <h3 className="text-gray-500">
```

Comment:
change this H3 to text-lg

Reply:
My bad, I meant to say change this H3 to H5!

Reply:
Can we also fix these two containers so they both fill the flex space and end up roughly the same width? 
<img width="1045" alt="Screenshot 2024-06-21 at 9 29 44 AM" src="https://github.com/sourcegraph/about/assets/102541294/f9a7d48c-55a4-4bb2-9b64-97e8a1ddc741">